app/src/main/res/values-fi/strings-fi.xml

```diff
@@ -204,10 +204,10 @@
 	<string name="wear_integration">Wear Integration</string>
 	<string name="send_data_to_android_wear_watch">Lähetä tietoja Android Wear Watchface.</string>
 	<string name="android_wear_integration">Android Wear Integration</string>
-	<string name="pref_wear_connectG5">Yhdistä Wear G5</string>
-	<string name="pref_summary_wear_connectG5">Yhdistä Wear suoraan G5</string>
-	<string name="pref_use_wear_connectG5">Käytä Wear G5 Collection Service</string>
-	<string name="pref_summary_use_wear_connectG5">Force puhelin käyttää Wear G5 Collection Service</string>
+	<string name="pref_enable_wearG5">Enable Wear Collection Service</string>
```

Comment:
Okay, then we should delete the new ones that I added.  I had also removed the old ones which were no longer being used, which was the original issue Jim E had mentioned - warnings about unused prefs, or something like that.